lochness/rpms/__init__.py

```diff
@@ -63,7 +96,7 @@ def initialize_metadata(Lochness: 'Lochness object',
 
     source_source_name_dict = {
         'beiwe': 'Beiwe', 'xnat': 'XNAT', 'dropbox': 'Drpbox',
-        'box': 'Box', 'mediaflux': 'Mediaflux',
```

Comment:
Sorry, why do we remove `mediaflux` entry?

Reply:
Is it because of these code below:

```python
        # mediaflux source has its foldername as its subject ID
        subject_dict = {
                'Subject ID': df_measure[rpms_id_colname].values,
                'Mediaflux': (f'mediaflux.{study_name}:' +
                              df_measure[rpms_id_colname]).values}
```

Reply:
Yes, since `mediaflux` ID is exactly same as the ID used in the `RPMS` table, I'm adding `Mediaflux` in a separate part as you've already noted.